src/components/CollectionAlgorithmDecision.js

```diff
@@ -30,6 +30,11 @@ export default function CollectionAlgorithmDecision(props) {
         div({style: {fontSize: '1.5rem'}}, [
           span({id: `collection-${id}-date-label`, style: {paddingRight: '1%'}}, ['Date:']),
           span({id: `collection-${id}-date-value`, style: {fontWeight: 400}}, [!isNil(createDate) ? formatDate(createDate) : 'N/A'])
+        ]),
+        div({style: {fontSize: '1.5rem'}}, [
+          span({id: `collection-${id}-reason-label`, style: {paddingRight: '1%'}}, ['Reason:']),
+          span({id: `collection-${id}-reason-value`, style: {fontWeight: 400}},
+            [!isNil(failureReasons) ? failureReasons.map((r) => span([r + ' '])) : 'N/A'])
```

Comment:
```suggestion
            !isArray(failureReasons) && failureReasons.length > 0 ? failureReasons.map((r) => span({ key: r }, [r + ' '])) : ['N/A']
```
like this?

Reply:
it would be just `isArray(failureReasons)` right, instead of `!isArray(...)`?

Reply:
oh yes! my brain 2 slow! Thank you!

Reply:
Same as above, use lodash's `isEmpty` method to do the nil and empty check for you

Reply:
Implemented isEmpty to do both checks, thank you!